utils/oomparser/oomparser_test.go

```diff
@@ -0,0 +1,204 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package oomparser
+
+import (
+	"os"
+	"testing"
+	"time"
+)
+
+const startLine = "Jan 21 22:01:49 localhost kernel: [62278.816267] ruby invoked oom-killer: gfp_mask=0x201da, order=0, oom_score_adj=0"
+const endLine = "Jan 21 22:01:49 localhost kernel: [62279.421192] Killed process 19667 (evilprogram2) total-vm:1460016kB, anon-rss:1414008kB, file-rss:4kB"
+const containerLine = "Jan 26 14:10:07 kateknister0.mtv.corp.google.com kernel: [1814368.465205] Task in /mem2 killed as a result of limit of /mem2"
+const containerLogFile = "containerOomExampleLog.txt"
+const systemLogFile = "systemOomExampleLog.txt"
+
+func createExpectedContainerOomInstance() (*OomInstance, error) {
```

Comment:
Maybe for both of these take the testing.T as an argument and do the t.Error() here. Will make the uses in tests simple since we don't have to check for errors there.